### PR TITLE
Xform sql feedback

### PR DESCRIPTION
**Pull request by benrudolph**
@snopoke this in response to the comments on the previous PR

cc: @proteusvacuum 
:blowfish:

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 7 errors:

* corehq/form_processor/utils/__init__.py, line 1 - F401 'should_use_sql_backend' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'acquire_lock_for_xform' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'adjust_datetimes' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'convert_xform_to_json' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'extract_meta_instance_id' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'new_xform' imported but unused
* corehq/form_processor/utils/__init__.py, line 13 - F401 'clean_metadata' imported but unused

**Comment by dimagimon** [comment]
There are 7 errors:

* corehq/form_processor/utils/__init__.py, line 1 - F401 'should_use_sql_backend' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'acquire_lock_for_xform' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'adjust_datetimes' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'convert_xform_to_json' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'extract_meta_instance_id' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'new_xform' imported but unused
* corehq/form_processor/utils/__init__.py, line 13 - F401 'clean_metadata' imported but unused

**Comment by dimagimon** [comment]
There are 7 errors:

* corehq/form_processor/utils/__init__.py, line 1 - F401 'should_use_sql_backend' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'acquire_lock_for_xform' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'adjust_datetimes' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'convert_xform_to_json' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'extract_meta_instance_id' imported but unused
* corehq/form_processor/utils/__init__.py, line 5 - F401 'new_xform' imported but unused
* corehq/form_processor/utils/__init__.py, line 13 - F401 'clean_metadata' imported but unused

File: corehq/ex-submodules/couchforms/tests/test_meta.py [hunk 1]
```diff
@@ -176,3 +172,11 @@ def testMetaDateInDatetimeFields(self):
 
         self.assertEqual(datetime(2014, 7, 10), xform.metadata.timeStart)
         self.assertEqual(datetime(2014, 7, 11), xform.metadata.timeEnd)
+
+    @run_with_all_backends
+    def test_missing_meta_key(self):
```
Comment (snopoke) [inline corehq/ex-submodules/couchforms/tests/test_meta.py]: What's the need for this test?

Comment (benrudolph) [inline corehq/ex-submodules/couchforms/tests/test_meta.py]: it was based on this comment: https://github.com/dimagi/commcare-hq/pull/8964#discussion_r44109703
tests still passed, but they should have failed

**Pull request merged**